### PR TITLE
SW-2109 Planting site database table

### PR DESCRIPTION
**Pull request by constanzauanini**
Screenshot 2022-11-11 at 17 37 07

Added search input UI but is not functional yet

### CODE REVIEW

File: src/components/PlantingSites/PlantingSitesCellRenderer.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export default function PlantingSitesCellRenderer(props: RendererProps<TableRowType>): JSX.Element {
+  const classes = useStyles();
+  const { column, row, value, index } = props;
+
+  if (column.key === 'name') {
+    return <CellRenderer index={index} column={column} value={value} row={row} className={classes.text} />;
+  }
+
+  return <CellRenderer {...props} className={classes.text} />;
```
Comment (karthikbtf) [inline src/components/PlantingSites/PlantingSitesCellRenderer.tsx:23]: line 23 does the same thing as line 20, we can maybe remove lines 19 - 21

File: src/components/PlantingSites/PlantingSitesList.tsx [hunk 2]
```diff
@@ -39,20 +53,47 @@ export default function PlantingSitesList(props: PlantingSitesListProps): JSX.El
...
       </PageHeaderWrapper>
       <Grid>
         <Box>
-          {searchResults?.length === 0 && (
+          {searchResults?.length === 0 ? (
             <Container sx={{ paddingY: 4 }}>
               <EmptyStatePage pageName={'PlantingSites'} />
             </Container>
+          ) : (
+            <PlantigSitesTable
```
Comment (karthikbtf) [inline src/components/PlantingSites/PlantingSitesList.tsx:91]: this is flashing before search finishes, we may need to hide this while search is happening

**Review by karthikbtf — APPROVED** [review]
looks good, some comments for follow up

File: src/components/PlantingSites/PlantingSitesCellRenderer.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/PlantingSites/PlantingSitesCellRenderer.tsx:23]: Yes, I just leave this prepared for adding the link to the name column, but I will remove it and add it back later

**Review by karthikbtf — APPROVED** [review]
🎉

**Pull request merged**